lib/handbook/addon/docs/components/new-project-modal/demo/template.hbs

```diff
@@ -0,0 +1,25 @@
+{{#docs-demo as |demo|}}
+        {{#bs-button class='center-block m-b-lg' onClick=(action this.openModal)}}
```

Comment:
Also wondering if it'd make sense to do like `<DeleteButton>` and include the button itself in the component. It'd guarantee consistent "create project" buttons and avoid repeating the show/hide logic. It also seems nice to have a single `<NewProjectButton>` invocation, rather than a button and modal at two different places in the template.

Reply:
I didn't do this one, since I didn't do the other refactors. I could potentially still do that.

Reply:
I think the main reason I haven't, aside from it slipping my mind, is that we currently have two invocations and those invocations both look different, so it's a weird time to make a component. The show/hide logic would still be nice to abstract, of course, but it could certainly wait.

Reply:
nvm. This is all getting refactored. I realized a dumb thing I did that changed the workflow, and that needs fixing, so I'll just get all this fixed.